.github/CODEOWNERS

```diff
@@ -1042,6 +1042,7 @@ packages/shared-ux/page/no_data/impl @elastic/kibana-global-experience
 packages/shared-ux/page/no_data/mocks @elastic/kibana-global-experience
 packages/shared-ux/page/no_data/types @elastic/kibana-global-experience
 packages/shared-ux/page/solution_nav @elastic/kibana-global-experience
+packages/shared-ux/prompt/errors @elastic/shared-ux
```

Comment:
It should be `@elastic/kibana-global-experience`

Reply:
Good catch. This was automatically done by CI.

@elastic/kibana-operations I guess it needs to be fixed there as well.

Reply:
I see you resolved the conversation, but I don't think the owner has changed?

Reply:
My git-fu failed me there for a second. Sorry about that